scripts/chains/smingot_chain.zs

```diff
@@ -0,0 +1,328 @@
+	.EUt(60)
+	.duration(200)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:dustCocoa>)
+	.fluidOutputs(<liquid:unsweetened_chocolate> * 144)
+	.EUt(180)
+	.duration(20)
+	.buildAndRegister();
+
+macerator.findRecipe(2, [<minecraft:dye:3>], []).remove();
+macerator.recipeBuilder()
+	.inputs(<ore:plateDensePressCakeGraded>)
+	.outputs(<ore:dustCocoa>.firstItem)
+	.EUt(120)
+	.duration(120)
+	.buildAndRegister();
+	
+mixer.recipeBuilder()
+	.inputs(<nuclearcraft:gelatin> * 2)
+	.fluidInputs(<liquid:albumen> * 500)
+	.fluidInputs(<liquid:sweetened_diluted_cane_syrup_mixture> * 5000)
+	.fluidOutputs(<liquid:marshmallow_syrup_mixture> * 5000)
+	.EUt(120)
+	.duration(300)
+	.buildAndRegister();
+
+assembler.recipeBuilder()
+	.inputs(<ore:plateDensePressCake>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 5000, 100)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+assembler.recipeBuilder()
+	.inputs(<ore:plateDensePressCakeDutched>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 7500, 300)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+metal_bender.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedMatterGrahamHot>)
+	.property("circuit", 0)
+	.outputs(<ore:plateDenseMatterGrahamHot>.firstItem)
+	.EUt(45)
+	.duration(80)
+	.buildAndRegister();
+	
+centrifuge.recipeBuilder()
+	.inputs(<ore:egg>)
+	.fluidOutputs(<liquid:albumen> * 100)
+	.fluidOutputs(<liquid:yolk> * 100)
+	.EUt(45)
+	.duration(60)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorPressed> * 2)
+	.outputs(<ore:plateDensePressCake>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorDutchedPressed> * 2)
+	.outputs(<ore:plateDensePressCakeDutched>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+furnace.addRecipe(<ore:crushedCentrifugedMatterGrahamHot>.firstItem, <ore:crushedCentrifugedMatterGraham>);
+
+var gelatins as IIngredient[] = [<ore:stickBone>, <ore:leather>, <minecraft:porkchop>, <minecraft:beef>, <minecraft:chicken>, <minecraft:rabbit>, <minecraft:mutton>];
+
+for gelatin_producer in gelatins {
+	extractor.recipeBuilder()
+		.inputs(gelatin_producer)
+		.outputs(<nuclearcraft:gelatin>)
+		.EUt(120)
+		.duration(100)
+		.buildAndRegister();
+}
+
+fluid_solidifier.recipeBuilder()
+	.fluidInputs(<liquid:fish_oil> * 400)
+	.inputs(<metaitem:shape.mold.plate>)
+	.outputs(<nuclearcraft:gelatin>)
+	.EUt(60)
+	.duration(100)
+	.buildAndRegister();
+	
+mixer.recipeBuilder()
+	.inputs(<ore:dustSugar>)
+	.inputs(<ore:dustWheat> * 3)
+	.inputs(<ore:dustSodiumBicarbonate>)
+	.fluidInputs(<liquid:butter> * 2000)
+	.fluidInputs(<liquid:milk> * 500)
+	.outputs(<ore:crushedCentrifugedMatterGraham>.firstItem * 8)
```

Comment:
That sodium bicarbonate should probably be recoverable.

Reply:
I don't think it really needs to be, as usually when cooking, you don't recover the baking soda from your recipe, as it doesn't easily come out, at least not without destroying what was going to be made in the end. I don't think I've ever recovered the baking soda while making a pancake, so neither would anyone do so here.